pkg/platform/model/vcs.go

```diff
@@ -344,6 +344,15 @@ func CommitHistoryPaged(commitID strfmt.UUID, offset, limit int64, auth *authent
 	return res.Payload, nil
 }
 
+func HasCommonParent(commit1, commit2 strfmt.UUID, auth *authentication.Auth) (bool, error) {
+	commonParent, err := CommonParent(&commit1, &commit2, auth)
+	if err != nil {
+		return false, errs.Wrap(err, "Could not get common parent")
+	}
+
+	return commonParent != nil, nil
+}
```

Comment:
I feel like this function is unnecessary, and that the user can simply call `model.CommonParent()` and assert that it is non-nil.